lbz/response.py

```diff
@@ -49,5 +69,21 @@ def to_dict(self) -> dict:
 
         return response
 
+    def json(self) -> dict:
+        if self._json is None:
+            if not isinstance(self.body, str):
+                raise ValueError(f"Got unexpected type to decode {type(self.body)}")
```

Comment:
Do we have to raise the `ValueError` on our own? It should be enough to let the `json.loads()` do its job 🤔